tests/unit/cluster/failover2.tcl

```diff
@@ -133,3 +133,101 @@ run_solo {cluster} {
         }
     } ;# start_cluster
 } ;# run_solo
+
+# start_cluster 3 1 {tags {external:skip cluster} overrides {cluster-replica-validity-factor 0}}
```

Comment:
```suggestion
```
I would still prefer, but I don't feel strongly.

Reply:
The proc only works in very specific pre-conditions and I think a comment about it would be good to have. I'd prefer a comment in plain English, but a code example is better than nothing IMHO.